.circleci/config.yml

```diff
@@ -247,6 +247,8 @@ workflows:
       - assume-role-development:
           context: api-assume-role-housing-development-context
           requires:
+            - run-tests
+            - sonar-scan
```

Comment:
sonar-scan step requires run-tests, as such here on line 250 requiring both the sonar scan, and its requirement run-tests is not needed.